### PR TITLE
Merge release_19.09 into dev

### PR DESCRIPTION
**Pull request by mvdbeek**
Not sure I resolved the admin job invocations correctly, can you have a look @dannon or @jmchilton ?

### CODE REVIEW

File: client/galaxy/scripts/components/admin/Jobs.vue [hunk 1]
```diff
@@ -129,8 +129,8 @@
                         :value="data.item['job_info']['id']"
                     ></b-form-checkbox>
                 </template>
-                <template v-slot:cell(job_info)="data">
```
Comment (nsoranzo) [inline client/galaxy/scripts/components/admin/Jobs.vue:132]: I didn't seen you also tried to merge 19.09 forward and opened https://github.com/galaxyproject/galaxy/pull/9104 . I resolved the conflict here differently, see https://github.com/galaxyproject/galaxy/pull/9104/files#diff-8fd44636765df0b938c8ff566954e688

File: lib/galaxy/tool_util/deps/mulled/util.py [hunk 2]
```diff
@@ -13,6 +14,9 @@
 
 log = logging.getLogger(__name__)
 
+BUILD_NUMBER_REGEX = re.compile(r'\d+$')
+MULLED_TAG_CACHE = collections.defaultdict(dict)
```
Comment (nsoranzo) [inline lib/galaxy/tool_util/deps/mulled/util.py]: Another difference, I removed this line since `MULLED_TAG_CACHE` is now unused.

Comment (mvdbeek) [reply lib/galaxy/tool_util/deps/mulled/util.py]: Ah yeah, I deleted that one now.

File: client/galaxy/scripts/components/admin/Jobs.vue (hunk 1 above)
Comment (mvdbeek) [reply client/galaxy/scripts/components/admin/Jobs.vue:132]: Ah, yeah, comparing that with dannon's PR, I think your variant is correct.

**Pull request closed without merging**